### PR TITLE
Define new workspace for pixel hetero comparison for online dqm

### PR DESCRIPTION
**Pull request by sroychow**
This PR adds a new workspace ```SiPixelHeterogeneous``` for the MEs related to GPU vs CPU comparison plots for pixel rechits, tracks and vertices. Test of the updates were done with tag ``` HG2207a```. The test GUI is running on lxplus787.cern.ch. To access, 
```
 ssh -NL 8070:localhost:8070 user@lxplus787.cern.ch
http://localhost:8070/dqm/online-dev/
```
Note:-
1D difference plots for vertices are missing from the harvested file (from run 353689). They should be available when HLT moves to CMSSW_12_3_5_patch1.

### CODE REVIEW

**Comment by sroychow** [comment]
Example plots
image

**Comment by sroychow** [comment]
@pmandrik thanks for the deployment, I checked the workspace and things look fine. I have swapped two plots for tracks and checked on the private gui on lxplus787 and it is working.

File: dqmgui/layouts/pixelHeterogenous-layouts.py [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+      'description': "Compare track z",
+      'draw': { 'withref': "no" }},
+     { 'path': "SiPixelHeterogeneous/PixelTracksGPUvsCPU/tip",
+      'description': "Compare track TIP [cm]",
+      'draw': { 'withref': "no" }},
+     { 'path': "SiPixelHeterogeneous/PixelTracksGPUvsCPU/nRecHits",
+      'description': "Compare number of rechits per track",
+      'draw': { 'withref': "no" }}],
+   [{ 'path': "SiPixelHeterogeneous/PixelTracksGPUvsCPU/nTracks",
+      'description': "Number of tracks (no qualiry cut)",
```
Comment (mmusich) [inline dqmgui/layouts/pixelHeterogenous-layouts.py:275]:
```suggestion
      'description': "Number of tracks (no quality cut)",
```

**Comment by pmandrik** [comment]
Hi, let me consider this question as a green light and start deployment of the PR to production online GUI.

**Pull request merged**